lib/src/util/io.dart

```diff
@@ -103,6 +120,36 @@ Stream<List<int>> sanitizeForWindows(Stream<List<int>> input) {
   });
 }
 
+/// Wraps [text] so that it fits within [lineLength], which defaults to
+/// [lineLength].
```

Comment:
which defaults to [_defaultLineLength]?

Reply:
Oops, it doesn't default at all anymore. Changed.